worlds/checksfinder/Regions.py

```diff
@@ -0,0 +1,64 @@
+
+def link_checksfinder_structures(world, player):
```

Comment:
This looks fairly overkill, as Entrance Randomization doesn't looks to be needed or supported.